2-mandatory/2-retrying-random-numbers.js

```diff
@@ -5,20 +5,23 @@
 
 // This function shouldn't be changed
 function generateRandomNumber() {
-    console.log("Generating number...");
-    return Math.round(Math.random() * 100);
+  return Math.round(Math.random() * 100);
 }
 
 function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
+  do {
+    let randomNum = generateRandomNumber();
+
+    if (randomNum > 50) return randomNum;
+  } while (1 < 52);
```

Comment:
`1 < 52` will always evaluate top `true`. While your solution works, I think the point of this exercise is to see if you can use the `while` clause to stop the loop once you've found a random number greater than 50. E.g.

```js
function getRandomNumberGreaterThan50() {
  let randomNum = 0;
  do {
    randomNum = generateRandomNumber();
  } while (randomNum <= 50);
  return randomNum
}
```

Reply:
thank you Yes I did not pay attention I just check and when all tests passed I went for next one.